proof/refine/ARM_HYP/Schedule_R.thy

```diff
@@ -486,44 +459,27 @@ crunch global_refs'[wp]: tcbSchedAppend valid_global_refs'
 crunch global_refs'[wp]: tcbSchedDequeue valid_global_refs'
   (wp: threadSet_global_refs)
 
-crunch irq_node'[wp]: tcbSchedEnqueue "\<lambda>s. P (irq_node' s)"
-  (simp: unless_def)
 crunch irq_node'[wp]: tcbSchedAppend "\<lambda>s. P (irq_node' s)"
   (simp: unless_def)
 crunch irq_node'[wp]: tcbSchedDequeue "\<lambda>s. P (irq_node' s)"
 
-crunch typ_at'[wp]: tcbSchedEnqueue "\<lambda>s. P (typ_at' T p s)"
-  (simp: unless_def)
 crunch typ_at'[wp]: tcbSchedAppend "\<lambda>s. P (typ_at' T p s)"
   (simp: unless_def)
-crunch typ_at'[wp]: tcbSchedDequeue "\<lambda>s. P (typ_at' T p s)"
 
-crunch ctes_of[wp]: tcbSchedEnqueue "\<lambda>s. P (ctes_of s)"
-  (simp: unless_def)
 crunch ctes_of[wp]: tcbSchedAppend "\<lambda>s. P (ctes_of s)"
   (simp: unless_def)
-crunch ctes_of[wp]: tcbSchedDequeue "\<lambda>s. P (ctes_of s)"
 
-crunch ksInterrupt[wp]: tcbSchedEnqueue "\<lambda>s. P (ksInterruptState s)"
-  (simp: unless_def)
 crunch ksInterrupt[wp]: tcbSchedAppend "\<lambda>s. P (ksInterruptState s)"
   (simp: unless_def)
 crunch ksInterrupt[wp]: tcbSchedDequeue "\<lambda>s. P (ksInterruptState s)"
 
-crunch irq_states[wp]: tcbSchedEnqueue valid_irq_states'
-  (simp: unless_def)
 crunch irq_states[wp]: tcbSchedAppend valid_irq_states'
   (simp: unless_def)
 crunch irq_states[wp]: tcbSchedDequeue valid_irq_states'
 
-crunch ct'[wp]: tcbSchedEnqueue "\<lambda>s. P (ksCurThread s)"
-  (simp: unless_def)
 crunch ct'[wp]: tcbSchedAppend "\<lambda>s. P (ksCurThread s)"
   (simp: unless_def)
-crunch ct'[wp]: tcbSchedDequeue "\<lambda>s. P (ksCurThread s)"
 
-crunch pde_mappings'[wp]: tcbSchedEnqueue "valid_pde_mappings'"
-  (simp: unless_def)
```

Comment:
lots of crunches went away here; removing un-needed ones, or something update related?

Reply:
removing unneeded ones. It's unclear if warnings are from the update or not, these specific ones probably are not, but I usually remove anything that looks feasible in an update instead of trying to figure out where it comes from.